lib/manager/components/transform/FeedTransformRules.js

```diff
@@ -0,0 +1,164 @@
+ */
+export default class FeedTransformRules extends Component<TransformRulesProps> {
+  _addTransformation = (type: string) => {
+    const {index, onChange, ruleSet} = this.props
+    const transformations = [...ruleSet.transformations]
+    transformations.push(newFeedTransformation(type))
+    onChange({...ruleSet, transformations}, index)
+  }
+
+  _removeRuleSet = () => {
+    const {index, onDelete} = this.props
+    const deleteConfirmed = window.confirm(
+      `Are you sure you would like to delete Transformation ${index + 1}?`
+    )
+    if (deleteConfirmed) onDelete(index)
+  }
+
+  _onChangeTransformation = (changes: any, transformationIndex: number) => {
+    const {index, onChange, ruleSet} = this.props
+    const transformations = [...ruleSet.transformations]
+    transformations[transformationIndex] = {...transformations[transformationIndex], ...changes}
+    onChange({...ruleSet, transformations}, index)
+  }
+
+  _onChangeRetrievalMethods = (options: Array<ReactSelectOption>) => {
+    const {index, onChange, ruleSet} = this.props
+    const retrievalMethods = options.map(o => o.value)
+    onChange({...ruleSet, retrievalMethods}, index)
+  }
+
+  _onToggleEnabled = (evt: SyntheticInputEvent<HTMLInputElement>) => {
+    const {index, onChange, ruleSet} = this.props
+    onChange({...ruleSet, active: !ruleSet.active}, index)
+  }
+
+  _removeTransformation = (transformationIndex: number) => {
+    const {index, onChange, ruleSet} = this.props
+    const transformations = [...ruleSet.transformations]
+    transformations.splice(transformationIndex, 1)
+    onChange({...ruleSet, transformations}, index)
+  }
+
+  _retrievalMethodToOption = (method: RetrievalMethod) => {
+    return {
+      value: method,
+      label: toSentenceCase(method.toLowerCase().split('_').join(' '))
+    }
+  }
+
+  render () {
+    const {feedSource, ruleSet, index} = this.props
+    const methodBadges = ruleSet.retrievalMethods.map(method =>
+      <VersionRetrievalBadge key={method} retrievalMethod={method} />)
+    return (
+      <ListGroupItem className={ruleSet.active ? '' : 'disabled'}>
+        <h4>
+          <ButtonToolbar className='pull-right'>
+            <Button
+              bsSize='xsmall'
+              onClick={this._onToggleEnabled}>
+              {ruleSet.active
+                ? <span><Icon type='pause' /> Pause</span>
+                : <span><Icon type='play' /> Resume</span>
+              }
+            </Button>
+            <Button
+              bsSize='xsmall'
+              onClick={this._removeRuleSet}>
+              <Icon type='trash' /> Delete
+            </Button>
+          </ButtonToolbar>
+          <Icon type='wrench' /> Transformation {index + 1}{' '}
+          {!ruleSet.active ? '(Paused) ' : ''}
+          <small>{methodBadges}</small>
+        </h4>
+        <small>
+          Apply this transformation to GTFS feeds created through the following methods.
+        </small>
```

Comment:
I think it'd be good to have some documentation or a link to more documentation for users to see here so they can learn what each of these transformation "methods" are.